tests/testthat/test-qenv_get_code.R

```diff
@@ -1,11 +1,15 @@
+get_code_g <- function(qenv, ...) {
+  gsub("\n", "", get_code(qenv, ...), fixed = TRUE)
+}
+
 testthat::test_that("get_code returns code (character by default) of qenv object", {
   q <- qenv() |>
     eval_code(quote(x <- 1)) |>
     eval_code(quote(y <- x))
-  testthat::expect_equal(get_code(q), paste(c("x <- 1", "y <- x"), collapse = "\n"))
+  testthat::expect_equal(get_code_g(q), c("x <- 1", "y <- x"))
```

Comment:
This is a breaking change. `get_code` used to return concatenated `character`.

Reply:
I think I did so many back and forths on this PR with so many features, that I just lost track. Thanks for catching
I reverted the output of `get_code` so it returns concatenated text in the same form as in input code https://github.com/insightsengineering/teal.code/pull/216/commits/7fa236931185c91dccc7df526696732991280625